osu.Game/Overlays/NotificationOverlay.cs

```diff
@@ -33,6 +34,9 @@ public partial class NotificationOverlay : OsuFocusedOverlayContainer, INamedOve
 
         public const float TRANSITION_LENGTH = 600;
 
+        public IEnumerable<Notification> AllNotifications =>
+            toastTray.Notifications.Concat(sections.SelectMany(s => s.Notifications));
```

Comment:
Not sure why the tests are failing on this and I can't reproduce locally but it does look a touch unsafe to access before loaded... Maybe this should say

```suggestion
        public IEnumerable<Notification> AllNotifications =>
            IsLoaded ? toastTray.Notifications.Concat(sections.SelectMany(s => s.Notifications)) : Array.Empty<Notification>();
```

or something.